src/celo/serializers.ts

```diff
@@ -126,6 +135,47 @@ function serializeTransactionCIP64(
   ]) as SerializeTransactionCIP64ReturnType
 }
 
+function serializeTransactionCIP66(
+  transaction: TransactionSerializableCIP66,
+  signature?: Signature | undefined,
+): SerializeTransactionCIP66ReturnType {
+  assertTransactionCIP66(transaction)
+
+  const {
+    chainId,
+    gas,
+    nonce,
+    to,
+    value,
+    maxFeePerGas,
+    maxPriorityFeePerGas,
+    accessList,
+    feeCurrency,
+    maxFeeInFeeCurrency,
+    data,
+  } = transaction
+
+  const serializedTransaction = [
+    toHex(chainId),
+    nonce ? toHex(nonce) : '0x',
+    maxPriorityFeePerGas ? toHex(maxPriorityFeePerGas) : '0x',
+    maxFeePerGas ? toHex(maxFeePerGas) : '0x',
+    gas ? toHex(gas) : '0x',
+    to ?? '0x',
+    value ? toHex(value) : '0x',
+    data ?? '0x',
```

Comment:
nit: at this point we're defining like a billion times `0x` in different serializers. using a const could be sensible

Reply:
introduced EMPTY_HEX_VALUE